src/ownership.tsx

```diff
@@ -85,7 +32,9 @@ export function get_owner_candidates(id: IdInfo, info: FileInfo) {
     // The last option is always available: create as the normal user.  In case
     // of something inside of the user's home directory, this was probably the
     // first option as well...
-    candidates.add(`${id.uid.name || id.uid.id}:${setgid || id.gid.name || id.gid.id}`);
+    // The first group from `cockpit.user()` is guaranteed to be the users default group
+    if (user.groups.length >= 1)
+        candidates.add(`${user.name || user.id}:${setgid || user.groups[0] || user.gid}`);
```

Comment:
This added line is not executed by any test.

Reply:
I guess coverage complains about `user.name` or `user.groups[0]` missing. That might be too tricky to test.

Reply:
the `|| user.gid` is redundant, as you already asserted that `user.groups[0]` exists. If that's empty (false-y), that's a bridge bug. So perhaps dropping that helps already. If it complains about the `|| user.id`, I'd just `// not-covered: OS error` that, it's not really reachable.

Reply:
This is a good case for an `else`.